src/OrchardCore.Build/Dependencies.props

```diff
@@ -40,6 +40,7 @@
     <PackageManagement Include="xunit.analyzers" Version="0.10.0" />
     <PackageManagement Include="xunit" Version="2.4.0" />
     <PackageManagement Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageManagement Include="YamlDotNet" Version="5.3.0" />
```

Comment:
Can you remove this library altogether, and parse the yaml manually. It should just be about reading line by line, and splitting on the name.

Reply:
Okay